docs/content/release_notes/whats_new.md

```diff
@@ -0,0 +1,21 @@
+# What's new in 1.8.0
+
+IBM® block storage CSI driver 1.8.0 introduces the enhancements that are detailed in the following section.
+
+**General availability date:** 16 December 2021
+
+## New HyperSwap support for IBM Spectrum Virtualize family storage systems
+
+IBM® block storage CSI driver 1.8.0 now supports HyperSwap implementation, when using IBM Spectrum Virtualize family storage systems.
+
+## New NVM Express (NVMe) over Fibre Channel host support for IBM Spectrum Virtualize family storage systems
+
+This version adds NVMe/FC support for supported IBM Spectrum Virtualize family storage systems using Red Hat® Enterprise Linux® (RHEL) operating systems. 
+
+## Increased StorageClass `SpaceEfficiency` parameter capabilities
+
+Version 1.8.0 increases the `SpaceEfficiency` deduplication parameter options for IBM Spectrum Virtualize family storage systems. For more information, see [Creating a StorageClass](../configuration/creating_volumestorageclass.md).
+
+## Additional support for Kubernetes 1.23 orchestration platforms for deployment
+
+This version adds support for orchestration platform Kubernetes 1.23, suitable for deployment of the CSI (Container Storage Interface) driver.
```

Comment:
same here, we are supporting K8S 1.21, K8S 1.22, OCP 4.7, OCP 4.8 and OCP 4.9 for 1.8.0

Reply:
updated for new ocp 4.9 (and removed 1.23)
done

Reply:
in the previous version we supported K8S 1.21, K8S 1.22 and OCP 4.8
in this version we support, in addition to the new support for OCP 4.9, we also support OCP 4.7 again (we supported OCP 4.7 in 1.6.0)

Reply:
correct but i don't think that we should list it as what's new because we had supported it in the past. it's there in the support documentation and on the matrix page. i believe that that is sufficient. 
@erantzabari  please confirm

Reply:
what about change log? re-supporting 4.7 is a 1.8.0 change (comparing to 1.7.0)

Reply:
added
done

Reply:
Agree, we don't need to add it to "what's new" section - we need to add it to the supported list